docs/getting-started/integrations/angular.md

```diff
@@ -113,15 +116,17 @@ The [`@ckeditor/ckeditor5-angular`](https://www.npmjs.com/package/@ckeditor/cked
 * [`@ckeditor/ckeditor5-utils`](https://www.npmjs.com/package/@ckeditor/ckeditor5-utils),
 * [`@ckeditor/ckeditor5-watchdog`](https://www.npmjs.com/package/@ckeditor/ckeditor5-watchdog).
 
-Keep in mind that they must have the same version as the editor build.
-
-Install all the required peer dependencies:
+Keep in mind that they must have the same version as the editor. Install all the required peer dependencies:
 
 ```bash
-npm install --save @ckeditor/ckeditor5-core @ckeditor/ckeditor5-engine @ckeditor/ckeditor5-utils @ckeditor/ckeditor5-watchdog
+npm install @ckeditor/ckeditor5-core @ckeditor/ckeditor5-engine @ckeditor/ckeditor5-utils @ckeditor/ckeditor5-watchdog
```

Comment:
Do we need to keep it? This is not needed with `ckeditor5`.

Reply:
Without those dependencies, I got typescript errors that "those types are not assignable to others." But I was testing earlier alpha. Let me check for the new version.

Reply:
Nope, for the newest nightly version, the errors still occurs. They occur in the template, and it is a type mismatch:

```Type 'typeof ClassicEditor' is not assignable to type '{ create(sourceElementOrData: string | HTMLElement, config?: EditorConfig | undefined): Promise<Editor>; }'```

```Type '{ toolbar: string[]; plugins: (typeof Bold | typeof Essentials | typeof Italic | typeof Paragraph | typeof Undo)[]; }' is not assignable to type 'EditorConfig'.```

Those dependencies are for the `@ckeditor/ckeditor5-angular` package. So, if we want to remove them, the integration itself would need to change, probably.

Reply:
Yep, we will change the integration, I would act as this would not be needed and let's remove this right now. We will review this as we have new versions of integrations.

Reply:
Ok, sure!